### DISCUSSION TITLE
Navigation arrows

### DISCUSSION THREAD

**Discussion by adm2k** [FreshRSS/FreshRSS#5484] [Q&A]
Is there any option to move the navigation arrows fron the left bottom to another location or maybe even add the to the "menu bar"?

![image](https://github.com/FreshRSS/FreshRSS/assets/120423301/f7b27ca2-83ab-4d6b-a3e0-187b3e68c1d6)

maybe here somewhere...
![image](https://github.com/FreshRSS/FreshRSS/assets/120423301/02a802da-459f-41d8-8635-8f114f861197)

**Comment by Alkarex** [comment]
There is an option to remove the navigation arrows. There is no option to change the location, but that can potentially be done in CSS, using the Custom CSS extension.

**Comment by adm2k** [comment]
Is this what you are referring to?
[FreshRSS-extensions](https://github.com/langfeld/FreshRSS-extensions/tree/master)
/xExtension-FixedNavMenu/

**Reply by Alkarex** [reply]
I was thinking more of setting your own CSS rules using https://github.com/FreshRSS/Extensions/tree/master/xExtension-CustomCSS
